File: 1 Periodo/Laboratorios/Laboratorio 2/Orientada a Objetos/soma.rb
=begin
Linguagem: Ruby
Paradigma = Orientada a Objetos (POO)
•SomaNumero - Programa que lê dois números do usuário e imprima a soma desses números na tela.
=end

puts "Escolha um valor para o 1º número: "
numero1 = gets.to_f#leitura da string com o gets, e conversão para float com o to_f

puts "Escolha um valor para o 2º número: "
numero2 = gets.to_f

soma = numero1 + numero2

puts "O valor da soma entre #{numero1} e #{numero2} resulta em #{soma}"

File: 1 Periodo/Atividades - Linguagens de Programacao/Procedural/soma.lua
--[[
Linguagem: Lua
Paradigma: Procedural
•SomaNumero - Programa que lê dois números do usuário e imprima a soma desses números na tela.
--]]

print("Escolha um valor para o 1º número: ")
local numero1 = io.read("*n") -- leitura da string com o io.read(), leitura de um número com o especificador "*n"
print("Escolha um valor para o 1º número: ")
local numero2 = io.read("*n")

local soma = numero1 + numero2

print(string.format("O valor da soma entre %g e %g resulta em %g", numero1, numero2, soma))
